src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionAnnotatedTypeFactory.java

```diff
@@ -194,6 +201,62 @@ public DependencyInjectionAnnotatedTypeFactory(BaseTypeChecker c) {
     this.postInit();
   }
 
+  /**
+   * Returns the string representation of the given type mirror. If the type mirror is a primitive
+   * or boxed type, the string representation is the type kind. Otherwise, the string representation
+   * is the fully-qualified name of the type.
+   *
+   * @param elementTypeMirror the type mirror
+   * @return the string representation of the given type mirror
+   */
+  protected static String resolveInjectionPointClassName(TypeMirror elementTypeMirror) {
```

Comment:
Why is the name of this parameter `elementTypeMirror` and not just `typeMirror`? I think this method should work for any type mirror.